src/parser/jobs/nin/index.tsx

```diff
@@ -16,12 +16,12 @@ export const NINJA = new Meta({
 	</>,
 
 	supportedPatches: {
-		from: '✖',
-		to: '✖',
+		from: '7.0',
+		to: '7.0',
```

Comment:
Makes sense; I can flip that back, especially since most of the module changes are me making assumptions and not actually concrete info from the math wizards.